src/app/cli/src/commands/system_api_server_run_command.rs

```diff
@@ -117,43 +115,52 @@ impl Command for APIServerRunCommand {
     async fn run(&mut self) -> Result<(), CLIError> {
         self.check_required_env_vars()?;
 
-        let access_token = self.get_access_token().await?;
-
-        // TODO: Cloning catalog is too expensive currently
```

Comment:
at the moment, it's not actual -- `Catalog` is `Arc<State>`, so this comment removed